effectiveOfficeBackend/src/main/kotlin/office/effective/common/notifications/FcmNotificationSender.kt

```diff
@@ -29,6 +45,58 @@ class FcmNotificationSender(private val fcm: FirebaseMessaging): INotificationSe
             .build()
         fcm.send(msg)
     }
+    
+    override fun sendUpdateContentMessages(topic: String, resourceId: String) {
+        logger.info("[fcmNotificationSender] Received update on {} calendar id", resourceId)
+        val workspace = calendarIdsRepository.findWorkspaceById(resourceId)
+        if (workspace.tag != "meeting") return
+        
+        val startTime = ZonedDateTime
+            .now(ZoneId.of(BookingConstants.DEFAULT_TIMEZONE_ID))
+            .toEpochSecond() * 1000
+        val endTime = startTime + ONE_WEEK_IN_MILLISECONDS
+        
+        val bookings = bookingService.findAll(
+            workspaceId = workspace.id,
+            bookingRangeFrom = startTime,
+            bookingRangeTo = endTime
+        )
```

Comment:
This method will retrieve participants with their integrations, although this data is ignored when sending the response. Integrations are retrieved by a separate database query, it would be better if we avoid it.